swan-lake/development-tutorials/observability/observe-ballerina-programs.md

```diff
@@ -25,15 +25,21 @@ This guide focuses on enabling Ballerina service observability with some of its
 <a href="https://prometheus.io/" target="_blank">Prometheus</a> and <a href="https://grafana.com/" target="_blank">Grafana</a> are used for metrics monitoring, and <a href="https://www.jaegertracing.io/" target="_blank">Jaeger</a> is used for distributed tracing. 
 
 Ballerina logs can be fed to any external log monitoring system like the 
-<a href="https://www.elastic.co/" target="_blank">Elastic Stack</a> to perform log monitoring and analysis.
+<a href="https://www.elastic.co/" target="_blank">Elastic</a> to perform log monitoring and analysis.
 
 ## Observe a Ballerina service
 
 Follow the steps below to observe a sample Ballerina service.
 
 ### Step 1 - create a `Hello World` Ballerina service
  
-Create a service as shown below and save it as `hello_world_service.bal`.
+Create a new Ballerina project using the `bal new` command.
+```shell
```

Comment:
```suggestion
```
```
Using `shell` results in incorrect syntax highlighting.

Reply:
Fixed in [78f7a57](https://github.com/ballerina-platform/ballerina-dev-website/pull/9401/commits/78f7a57ec960e4243cf97f4640ba4bd24f2377ae)